examples/functions-with-defaultable-parameters/functions_with_defaultable_parameters.bal

```diff
@@ -16,24 +16,36 @@ function printSalaryDetails(int baseSalary,
 }
 
 public function main() {
-    // Calls the function by passing only the `baseSalary`.
-    // The `annualIncrement` and the `bonusRate` defaults to 20 and 0.02
+    // Call the function by passing only the `baseSalary`.
+    // The `annualIncrement` and the `bonusRate` default to 20 and 0.02
     // respectively.
     printSalaryDetails(2500);
 
-    // Calls the function by passing only the `baseSalary` and `annualIncrement`.
-    // Passes the defaultable parameter as a key-value pair when invoking the
-    // function. The `bonusRate` defaults to 0.02.
+    // Call the function by passing only the `baseSalary` and `annualIncrement`.
+    // `annualIncrement` is passed as a named argument.
+    // The `bonusRate` defaults to 0.02.
     printSalaryDetails(2500, annualIncrement = 100);
 
-    // Calls the function by passing only the `baseSalary` and `bonusRate`.
+    // Call the function by passing only `baseSalary` and `annualIncrement`
+    // again, but now pass `annualIncrement` as a positional argument.
```

Comment:
```suggestion
    // again, but now the `annualIncrement` is being passed as a positional argument.
```

Reply:
Is this correct? The sentence starts with "Call".

Reply:
The instructions are not clear when we say "but now pass". Maybe, we need to reword.

Reply:
This is now rephrased with the changes to the file.